### DISCUSSION TITLE
[Feb 08] sharing gallery view

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1269] [Discord Threads]
> **gab696** _queried at_ ```2022-02-06T20:52:05.921000+00:00```  
## HI ! I can share forms and grid views but not the gallery  
  
> **o1lab** _commented at_ ```2022-02-06T21:59:00.826000+00:00```  
#### Currently not possible  
  
> **gab696** _commented at_ ```2022-02-07T07:48:56.898000+00:00```  
#### 😭 thanks  
  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/939986836451905556)  
 
 - - - -

#### Join [NocoDB's](http://www.nocodb.com/)  community
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by esteveaq** [comment]
Would be interested for this functionality ! Extremely useful for sharing/ communication purposes

**Comment by lcfd** [comment]
I'm trying to share a Gallery too. But the share view button appears only for grid and forms.
Is it the expected behavior?

**Reply by o1lab** [reply]
Share on Gallery is yet to be implemented.

**Reply by lcfd** [reply]
Ok, good to know. Thank you!
Just out of curiosity, what's different than the grid view share?

**Reply by o1lab** [reply]
Previously, it was not very similar. Now it is - we will get to it once we've [vue3](https://github.com/nocodb/nocodb/projects/4) sprint done :)

**Reply by lcfd** [reply]
Cool! I can't wait to see it! 🎉
I'm trying to become gradually a pro user of this tool.
I can do whatever I want just by using some code but I wanna discover where the limits of the platform are.

Thank you again 😊
